family2family/build.gradle

```diff
@@ -11,13 +10,18 @@ buildscript {
     }
 }
 
+// Update appropriately for releases
+version = '1.0.0-SNAPSHOT'
+
 apply plugin: 'java'
 apply plugin: 'idea'
 apply plugin: 'org.springframework.boot'
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
+jar.archiveName = "family2family.${jar.extension}"
```

Comment:
Not from that actually...I added a "version = 1.0.0-SNAPSHOT" variable to the script.  Because I added the version, it's appending to the jar name.